### ISSUE TITLE
AVT 3 - Tabs: Tab content of tab 1 and 2 are not announced by VoiceOver

### ISSUE THREAD

**Issue by snidersd** [IBM/carbon-components-react#2223]
## Environment
macOS Mojave version 10.14.2
Chrome version 72.0.3626.96 (Official Build) (64-bit)

## Detailed Description
Cannot navigate using the keyboard with Voiceover to tab 1 or tab 3 content. Tab 4 content is announced only after VoiceOver announces it has left the Tab navigation. Also, see issue #2171. 

## Expected Result
VO should announce each tab using the Right and Left Arrow keys. After a Tab is selected the Tab key should navigate to the tab content, as well as tab back to the original tab. See ARIA Authoring Practices 1.1 - Tabs for additional ARIA requirements.

**Comment by snidersd** [comment]
This issue should remain open until it is fixed and retested.
